docs/release/upgrade.adoc

```diff
@@ -45,6 +45,21 @@ When upgrading to new feature versions of Enonic XP, new versions of standard ap
 After upgrading XP, check for new versions of your installed applications from Enonic market. 
 We generally always recommend updating your apps to the latest version!
 
+== v7.11 notes
+
+WARNING: FULL RESTART REQUIRED - For security reasons, XP's clustering engine (Hazelcast) has been updated in this release.
+A full restart is required in order to prevent loss of data or state.
+
+* Virtual application schemas may now override schema present in any installed (real) application.
+To prevent virtual applications from overriding use config
```

Comment:
This looks like an unfinished sentence... `config` should probably be a link, followed by ".".